spcs-basic/README.md

```diff
@@ -0,0 +1,73 @@
+# Instructions
+
+## Set your SnowCLI connection (optional)
```

Comment:
Let's add a section at the top for requirements (docker / docker buildx for example)

Reply:
Done!